src/reference/00-Getting-Started/02-sbt-by-example.md

```diff
@@ -126,15 +133,6 @@ Hello
 [success] Total time: 1 s, completed May 6, 2018 4:10:44 PM
 ```
 
-### Switch Scala version from shell
-
-```
-sbt:foo-build> ++2.13.0-M3!
-[info] Forcing Scala version to 2.13.0-M3 on all projects.
-[info] Reapplying settings...
-[info] Set current project to foo-build (in build file:/private/tmp/foo-build/)
-```
```

Comment:
Let's keep this and maybe remove the two sections below that uses `session save`, which not really used much.

Reply:
I would suggest that the `++` section is moved to later in the text; as it is, the user can see the first `set` and understand that it sets something, and the rest follows quite logically.

Reply:
Besides, `++` is temporary, and it is mainly only used once you have cross-building.

Reply:
Cross building is definitely a feature we should show case early, since it's a frequently used feature of sbt. Moving to later section might make sense actually tho.